app/customer.py

```diff
@@ -0,0 +1,47 @@
+from math import sqrt
```

Comment:
The import statement for `sqrt` is incorrect. It should be `from math import sqrt` instead of `from math import sqrt /`.